src/plugins/intel_npu/src/plugin/npuw/compiled_model.cpp

```diff
@@ -385,13 +385,13 @@ ov::npuw::CompiledModel::CompiledModel(const std::shared_ptr<ov::Model>& model,
             m_compiled_submodels[id].host_gather = subgraph._host_gather;
             m_compiled_submodels[id].quant_unpack_gather = subgraph._quant_unpack_gather;
             m_compiled_submodels[id].param_base = fcn_template._param_offset;
-            m_compiled_submodels[id].closure = subgraph._closure;
+            m_compiled_submodels[id].closure.get_closure() = subgraph._closure;
```

Comment:
this semantic is a bit broken btw.

`.get_closure()` should lock until the bank is evaluated. It is not even the case here.

I believe `unsafe_get()` is what you need here.

Reply:
It's fine. In regular `get` we skip the `future` check since it's not yet initialized == invalid